### PR TITLE
chore: update NPM dependencies

### PR DESCRIPTION
**Pull request by TimvdLippe**
As reported in b/158734154 for Google-internal.

### CODE REVIEW

**Comment by TimvdLippe** [comment]
Note that the tests appear to be failing, but I honestly have no clue why :( Travis also doesn't seem to be running the tests, so I don't know what could have gone wrong. This is the failure message:

```
 FAIL  src/WasmDis.spec.ts
  ● NameSectionReader › Wasm module with bad names

    parseWat failed:
    test.wat:3:11: error: redefinition of function "$foo"
             (import "import" "function2" (func $foo))
              ^^^^^^

      251 | 
      252 |   test("Wasm module with bad names", () => {
    > 253 |     const { buffer } = parseWat(
          |                        ^
      254 |         `test.wat`,
      255 | 	`(module
      256 |          (import "import" "function" (func $foo))

      at parseWat (node_modules/wabt/index.js:26:14344)
      at Object.<anonymous> (src/WasmDis.spec.ts:253:24)

 PASS  ./test.ts
 PASS  ./wabt-compatibility.spec.ts

Test Suites: 1 failed, 2 passed, 3 total
Tests:       1 failed, 876 passed, 877 total
Snapshots:   0 total
Time:        2.849 s, estimated 6 s
```

**Comment by TimvdLippe** [comment]
Test failures are related to the upgraded version of `wabt`. Downgrading to 1.0.13 fixes the issue. The regression is therefore in `wabt@1.0.15` or `wabt@1.0.14` (although the 1.0.14 is not published on NPM). Release notes are at https://github.com/WebAssembly/wabt/releases/tag/1.0.15

Likely an issue/change in Emscripten?

**Comment by yurydelendik** [comment]
> Travis also doesn't seem to be running the tests

I think I address that in #63 -- can you re-push this branch?

**Comment by TimvdLippe** [comment]
Done.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 node_modules/
 dist/
 npm-debug.log
+esm/
```
Comment (bmeurer) [inline .gitignore:5]: Why is this needed?

Comment (TimvdLippe) [reply .gitignore:5]: `npm run build` generates this folder

Comment (bmeurer) [reply .gitignore:5]: It doesn't on `master`, and it shouldn't generate this folder. Looks like the update breaks something. What's in there?

Comment (TimvdLippe) [reply .gitignore:5]: Hm, might have been a local issue. I will check tomorrow.

Comment (TimvdLippe) [reply .gitignore]: Fixed.
